spec/unit/resources/ethernet_network/create_spec.rb

```diff
@@ -27,3 +27,14 @@
     expect(real_chef_run).to create_oneview_ethernet_network('EthernetNetwork1')
   end
 end
+
```

Comment:
Agreed. This is just a sample of what you would do if you wanted to test API300 resources. I think this particular one is probably OK to leave in since it's the only one so far, but we wouldn't have to add these sort of tests for other resources that are the same as API200.